ue5osc/__init__.py

```diff
@@ -100,6 +108,10 @@ def set_quality(self, graphics_level: int) -> None:
         """Set the graphics quality level from 0 (low) to 4 (high)."""
         self.client.send_message("/set/quality", graphics_level)
 
+    def set_texture(self, object: int, material: int) -> None:

```

Comment:
Instead of object being an `int`, we could instead create an enum class with the variants of wall, floor, ceiling. The variant indices must also match Unreal.

Reply:
From my understanding, you would like something like this.
```
class ObjectType(IntEnum):
    FLOOR = 0
    WALL = 1
    CEILING = 2
```
To go along with this I edited the way we randomly change the texture in boxnav.